wsgi-conf.ini

```diff
@@ -5,3 +5,4 @@ master = true
 processes = 5
 
 socket = 0.0.0.0:5100
+http = 0.0.0.0:5000
```

Comment:
apparently there was nothing really running on port 5000, and when I tried to use port 5100 I was getting this error: https://stackoverflow.com/questions/15878176/uwsgi-invalid-request-block-size

Reply:
This is the production configuration to be used with nginx. therefor http is not necessary and port 5000 is still usable through the docker-compose.yml directly linking to the flask api

Reply:
I just noticed the problem, this actually doesn't work:
```
CMD python server.py runserver -h 0.0.0.0
CMD uwsgi --ini wsgi-conf.ini
```
on docker we can only ever have a single CMD, if we put two docker doesn't run them both, instead the last one overrides all others

so I'll change it to this:
`CMD uwsgi --ini wsgi-conf.ini & python server.py runserver -h 0.0.0.0`
this way we have uwsgi on the background and python flask on the foreground